guardduty_notification.tf

```diff
@@ -66,7 +66,7 @@ resource "aws_lambda_function" "guardduty_notification_lambda" {
   filename         = data.archive_file.zip.output_path
   source_code_hash = data.archive_file.zip.output_base64sha256
   role             = aws_iam_role.guardduty_notification_iam_role[count.index].arn
-  handler          = "guardduty_notification.lambda_handler"
+  handler          = "guardduty_notification.lambda_handler-${var.aws_region}"

```

Comment:
@tatamarques I think it will break the lambda as you are changing the handler.

Reply:
@brunodasilvalenga  But it won't work for multi-region accounts. If the handler exists in one region, when we create it in another region, we get a conflict 😏. Any idea how we can fix it?

Reply:
You didn't get it. I'm saying that you are changing the handler of the lambda function.
The handler should be a file and a function, in this case you are changing it to be dynamic but inside of the lambda function your function is not dynamic.

ref: https://github.com/DNXLabs/terraform-aws-guardduty/blob/master/guardduty_notification.py#L8